Store/dataobjects/StoreOrderItem.php

```diff
@@ -403,6 +403,17 @@ protected function getSerializablePrivateProperties()
 		return $properties;
 	}
 
+	// }}}
+	// {{{ protected function getFinalPrice()
+
+	public function getFinalPrice()
+	{
+		$promotion_discount =
+			$this->promotion_discount ? $this->promotion_discount : 0.00;
```

Comment:
`promotion_discount` does not exist as a property in this package. Do you mean `discount`? If not, this code may need to live in a different package.

Reply:
You are correct it lives in `promo`, that was an oversight. I will update accordingly.